apps/src/blockly/addons/cdoUtils.js

```diff
@@ -108,3 +108,12 @@ export function getCode(workspace) {
   // After supporting JSON block sources, change to:
   // return JSON.stringify(Blockly.serialization.workspaces.save(workspace));
 }
+
+// TODO: Re-define with a new custom field.
+export function playSoundButton(dashboard, onSelect) {
+  return new Blockly.FieldDropdown([['Choose', 'Choose']], () => {
+    dashboard.assets.showAssetManager(onSelect, 'audio', null, {
+      libraryOnly: true,
+    });
+  });
```

Comment:
While this looks right now like duplicated code, it gives us a great place experiment with new fields for the Google Blockly version without risk of causing regressions in Sprite Lab using CDO Blockly! :tada:

Reply:
Thanks so much for pairing with me on this!

Reply:
Is it possible to set this up this function to not require passing in all of dashboard as an argument? It seems like the two arguments are used to set up an `onChange`, right?

Could you set up something where you're calling this function like
```
const onChange = () => 
    dashboard.assets.showAssetManager(onSelect, 'audio', null, {
      libraryOnly: true,
    });
Blockly.cdoUtils.playSoundButton(onChange);
```

Reply:
Is [this change](https://github.com/code-dot-org/code-dot-org/pull/51515/commits/4efe5c5937b74cf3cff97434c7faa46ecf6af1cf) what you mean?
For some reason, when I tested locally, the sound picker modal is displayed immediately before clicking on the field.

Reply:
https://user-images.githubusercontent.com/107423305/234408429-b7c65cad-5430-4cd1-8af3-4708368a910d.mp4

Reply:
Yep that's pretty much it! I think you just need to put the contents of onChange into the body of an arrow function, so that it can be called (right now it's being called when you define it)

Reply:
Thanks so much! It may end up changing in follow-up but this is much cleaner for now.